### ISSUE TITLE
Warlock pet bar missing Aggressive and Defensive buttons

### ISSUE THREAD

**Issue by apav** [Wall-core/Everlook-Bugtracker#729]
My warlock's pet bar is missing the following pet bar buttons for each of my demons:

Voidwalker - missing Aggressive and Defensive
Succubus  - missing Aggressive and Defensive

Example for Succubus:

![missing pet buttons](https://user-images.githubusercontent.com/8137930/204952103-9d1f5b84-7b27-4c2f-a33a-6d928c99fbc9.png)

This is an issue because AFAIK there's no way to add them back to the pet bar since they are not in the spellbook. 

The Imp has them all and I haven't unlocked any other pet beyond Succubus yet. I don't remember if I ever had those buttons on my action bar when I first unlocked the Voidwalker and Succubus, but I've hardly used my Imp at all since getting my Voidwalker so that might explain why the Imp's bar is intact. If this bug didn't happen when I first unlocked these demons, I'm not sure what caused it. You normally can't remove these buttons off your bar.

**Comment by Godmatik** [comment]
Does this still happen if you have all addons turned off?

**Comment by apav** [comment]
> Does this still happen if you have all addons turned off?

Yes, this screenshot was taken with all addons disabled and the WDB folder deleted.

**Comment by fleksnes** [comment]
This must be a client or an account issue. You are right about that you should not be able to remove the "stances" from the pet bar (although you can move them to a differnt slot on the pet bar), and that they are not in the spell book.  Make sure your client is a clean 1.12.1 (5875) client. Not Turtle wow for example. If it is, then try to disable all addons, close wow, delete WBD folder and try to move the WTF folder out of the wow folder (this way you have a copy of it). The WTF folder contains all settings such as keybinds, macro, addon settings etc. Now try to open wow and see if its there. If it still does not work try to ask for help in support channel in Discord.

**Comment by fleksnes** [comment]
I tested and the location of spells on your action bars is stored server-side. So you might have to get help from a GM/dev. If what I said does not work, as in support channel make/ an in game ticket.

**Comment by Gamemechanicwow** [comment]
Curious. Did you ever get this solved?

**Comment by apav** [comment]
> Curious. Did you ever get this solved?

Unfortunately not. It's something on the server side so there's nothing I can do. One of the GMs said they'd look into my account when I was offline months ago, but I'm not sure if they ever did because I never heard any updates about it.

**Comment by apav** [comment]
@Gamemechanicwow Do you have any idea what might fix this issue? It is still persisting and GMs are unwilling to try a character copy (and delete original). Maybe deleting the summon demon spells and readding them? Or forcing the demons to be abandoned, generating new ones with different names?
